pressbooks/pressbooks
# Improve dashboard display

Minor fixes for create and clone a book buttons on user dashboard. Fix for https://github.com/pressbooks/pressbooks/issues/2646

To test:
Visit user dashboard and check that the create a book and clone a book links appear next to each other and display properly at all screen widths:
![Screenshot from 2022-05-19 16-06-27](https://user-images.githubusercontent.com/13485451/169418565-cfe6cfc6-3ef8-4df1-afbb-97360ef7a349.png)
